### ISSUE TITLE
Multi-step fine tuning

### ISSUE THREAD

**Issue by AdrienAudren** [meteofrance/py4cast#154]
![Image](https://github.com/user-attachments/assets/18f2c50a-b01c-4a8f-89a2-228c29f05acf)

We progressively add one forecast step to the backpropagation chain (see backpropagation through time BPTT). Figure above shows results average over 1 month of test data with 45 lead times rollouts for total precipitation weather variable and BPTT=9. We usually do this by deactivating the scheduler and setting the learning rate to its minimum value reached at end of first training stage, which is BPTT=1. One epoch is enough (maybe too many batches ?) for each forecast step we add during fine tuning stage.
